src/App.tsx

```diff
@@ -8,7 +7,58 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const [select, setSelect] = useState('all');
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+  const [allTodos, setAllTodos] = useState<Todo[]>([]);
+  const [selectTodo, setSelectTodo] = useState<Todo>();
```

Comment:
Add a null to the type here, in case you have a situation when the todo can be null
```suggestion
  const [selectTodo, setSelectTodo] = useState<Todo | null>(null);
```